test/ai/assistant.test.ts

```diff
@@ -50,7 +50,7 @@ describe.skip('assistant', () => {
   });
 
   it('should list assistants', async () => {
-    const result = await ainft.assistant.list(objectId, null);
+    const result = await ainft.assistant.list([objectId], null);
```

Comment:
objectIds?

Reply:
oh! it's a constant value, maybe `objectIds=[objectId]` thanks!